client/my-sites/plugins/access-control.js

```diff
@@ -61,6 +61,20 @@ const getMockBusinessPluginItems = () => {
 	} );
 }
 
+const getWpcomPluginPageError = ( siteSlug ) => {
+	return {
+		title: i18n.translate( 'Want to add a store to your site?' ),
+		line: i18n.translate( 'Support for Shopify, Ecwid, and Gumroad is now available for WordPress.com Business.' ),
+		action: i18n.translate( 'Upgrade Now' ),
+		actionURL: '/plans/' + siteSlug,
```

Comment:
What if `siteSlug` is `undefined`, like in [this line](https://github.com/Automattic/wp-calypso/pull/2562/files#diff-04f267a2444876241c80473da9d90acbR123)?